src/nnfusion/engine/profiler/cuda_runtime.cpp

```diff
@@ -68,7 +71,18 @@ bool CudaDefaultRuntime::codegen(const ProfilingContext::Pointer& ke)
     writer << "\n";
     for (auto& it : re->local_symbol)
         if (it.second->symbol.find("declaration::") != string::npos)
+        {
+            // if (it.second->symbol == "declaration::ort_softmax" || it.second->symbol == "declaration::cuda_layer_norm")
```

Comment:
remove these comments if they are not useful?

Reply:
sure